demisto_sdk/commands/validate/validators/IF_validators/IF115_unsearchable_key.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+
+from typing import Iterable, List
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    ValidationResult,
+)
+
+ContentTypes = IncidentField
+
+
+class UnsearchableKeyValidator(BaseValidator[ContentTypes]):
+    error_code = "IF115"
+    description = "Checks if the `unsearchable` key set to true."
+    rationale = (
+        "Marking many items searchable causes index and search loads on the platform. "
+        "Official demisto/content IncidentField files must be set to Unsearchable. "
+        "In custom content, it's recommended to keep the number of searchable fields to a minimum."
+    )
+    error_message = (
+        "Warning: Incident fields should have `unsearchable` set to true. "
```

Comment:
@YuvHayun is this how we do warnings in the new format?

Reply:
Nope, if we want this validation to only throw warnings then the error code should be under warning in the validation config.
But since this validation only runs on added files, I think it's best that we throw error and force merge if needed.